docs/snippets/_mobile_docs_prebid-mobile.mdx

```diff
@@ -108,28 +36,17 @@ To configure your UID2 Prebid for Mobile integration, follow the steps below.
    </dependency>
    ```
    </TabItem>
-   <TabItem value='ios' label='iOS'>
-   #### Installing with Package.swift
-   Add the following dependency to Package.swift:
+   <TabItem value='cocoapods' label='CocoaPods (iOS)'>
+   Add the following entry in your `Podfile`:
    ```js
-   dependencies: [
-      .package(url: "https://github.com/IABTechLab/uid2-ios-sdk.git", from: "1.2.0"), // TODO change
-   ]
-   ```
-   #### Installing with Xcode
-   In the XCode user interface, under Package Dependencies, add the following entry for your apps:
-   | Name         | Location                                     | Dependency Rule                         |
-   | :----------- | :------------------------------------------- |:----------------------------------------|
-   | uid2-ios-sdk | `git@github.com:IABTechLab/uid2-ios-sdk.git` | Up to next major version: 1.2.0 < 2.0.0 |
-
-   #### Installing with CocoaPods
-
-   Add the following entry in your `Podfile`:
-
-   ```
-   pod 'UID2', '~> 1.2' // TODO change
+   pod 'UID2Prebid', '~> 1.5'
    ```
    </TabItem>
+   <TabItem value='spm' label='SPM (iOS)'>
+   :::warning
+   Integration via Swift Package Manager is not supported until it is supported by the Prebid Mobile SDK itself. See [Prebid's Mobile SDK documentation](https://docs.prebid.org/prebid-mobile/pbm-api/ios/code-integration-ios.html#swift-pm) for more information.
```

Comment:
Just turn it around. Plus in tech docs we're instructed to say "for details" (internal style guide decision):

For details, see [Prebid's Mobile SDK documentation](https://docs.prebid.org/prebid-mobile/pbm-api/ios/code-integration-ios.html#swift-pm).